test/suites/ipv6/suite_test.go

```diff
@@ -94,4 +97,18 @@ var _ = Describe("IPv6", func() {
 		})
 		Expect(internalIPv6Addrs).To(HaveLen(1))
 	})
+	It("should provision a static IPv6 prefix with node launch and set IPv6 as primary in the primary network interface", func() {
+		output, err := env.EC2API.DescribeInstances(env.Context, &ec2.DescribeInstancesInput{
+			InstanceIds: []string{coretest.Pod().Spec.NodeName},
+		})
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(output.Reservations[0].Instances[0].NetworkInterfaces[0].Ipv6Prefixes).To(HaveLen(1))
```

Comment:
Does checking the IPv6 prefix value == 1 tell us anything? Since the CNI can set this as well, I get that this is validating _that_ it gets set, but I'm not sure that this validates that Karpenter is actually the thing that sets it

Reply:
I think unfortunately this is hard to be checked for sure from an automation. CNI would skip provision a prefix is the prefix has been added into LT. But tests may not be able to know. Unless we can somehow set CNI not to provision.